### PR TITLE
Seller assessment flow

### PR DESCRIPTION
**Pull request by ndavison**
This PR adds the new seller domain assessment flow based on the collection of 'evidence' against domain selection criteria. This covers both the new front end components (for when a seller instigates the assessment), and the admin portal components when assessing submissions.

### CODE REVIEW

File: apps/marketplace/components/SellerDashboard/Categories.js [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+            <a href={`${rootPath}/seller-assessment/create/${category.id}`}>Resubmit</a>
+          </React.Fragment>
+        )
+      case 'submitted':
+      default:
+        return ''
+    }
+  }
+
+  categoryIdToHash = domainId => {
```
Comment (malmas18) [inline apps/marketplace/components/SellerDashboard/Categories.js]: helper function?

File: apps/marketplace/pages/SellerEditFlowPage.js [hunk 2]
```diff
@@ -122,10 +122,8 @@ export class SellerEditFlowPage extends Component {
         saveModel={this.save}
         returnPath={`${rootPath}/seller-dashboard`}
         previewPath={`${rootPath}/digital-marketplace/opportunities/${supplierCode}`}
-        progressButtons={{
-          startText: 'Save & continue',
-          showReturnText: false
-        }}
+        showReturnButton={false}
```
Comment (malmas18) [inline apps/marketplace/pages/SellerEditFlowPage.js]: didn't want to categorise the some properties?

File: src/bundles/ApplicationsAdmin/components/EvidenceAssessment/EvidenceAssessment.js [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+    const evidenceId = this.props.match.params.id
+    this.props.approveEvidence(evidenceId).then(res => {
+      this.setState({
+        wasApproved: true,
+        wasRejected: false
+      })
+    })
+  }
+
+  priceIsOver() {
```
Comment (malmas18) [inline src/bundles/ApplicationsAdmin/components/EvidenceAssessment/EvidenceAssessment.js]: small thing.. do we allow decimals in the price?

File: src/bundles/ApplicationsAdmin/components/EvidenceAssessmentList/EvidenceAssessmentList.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+              </tr>
+            </thead>
+            <tbody>
+              {evidence.map(e => (
+                <tr key={e.id}>
+                  <td>{format(e.submitted_at, 'DD-MM-YYYY')}</td>
+                  <td>{e.supplier_name}</td>
+                  <td>
+                    {e.brief_id && (
+                      <a href={`https://marketplace.service.gov.au/2/digital-marketplace/opportunities/${e.brief_id}`}>
```
Comment (malmas18) [inline src/bundles/ApplicationsAdmin/components/EvidenceAssessmentList/EvidenceAssessmentList.js]: hard coded url?

File: src/bundles/ApplicationsAdmin/components/EvidenceAssessment/EvidenceAssessment.js (hunk 3 above)
Comment (ndavison) [reply src/bundles/ApplicationsAdmin/components/EvidenceAssessment/EvidenceAssessment.js]: I don't believe so (current profile edit pricing doesn't allow it).

File: src/bundles/ApplicationsAdmin/components/EvidenceAssessmentList/EvidenceAssessmentList.js (hunk 4 above)
Comment (ndavison) [reply src/bundles/ApplicationsAdmin/components/EvidenceAssessmentList/EvidenceAssessmentList.js]: There is no straight forward way from the admin frontend to detect which environment it is running in from what I could tell (e.g. even if we can detect it isn't prod, how do we know which of our rc/dev/alpha etc environments is it?)

Comment (ndavison) [reply src/bundles/ApplicationsAdmin/components/EvidenceAssessmentList/EvidenceAssessmentList.js]: Updated to use the env var value.

**Pull request merged**